src/lib.rs

```diff
@@ -473,6 +473,13 @@ pub type Ixs = isize;
 /// [`.slice_move()`]: #method.slice_move
 /// [`.slice_collapse()`]: #method.slice_collapse
 ///
+/// It's possible to take multiple simultaneous *mutable* slices with the
+/// [`.multi_slice_mut()`] or (for [`ArrayViewMut`] only)
+/// [`.multi_slice_move()`].
+///
+/// [`.multi_slice_mut()`]: #method.multi_slice_mut
+/// [`.multi_slice_move()`]: type.ArrayViewMut.html#method.multi_slice_move
```

Comment:
I don't know about you, but I don't mind transitioning to nightly-only rustdoc links (because they render correctly in docs.rs).. Importance of rendering the links when the user generates docs on stable?

Reply:
That would be fine, although I'm having trouble getting nightly-style links to work correctly. The following don't work:

```markdown
[`.multi_slice_mut()`]
```
```markdown
[`multi_slice_mut()`]
```

If I add an explicit destination, `multi_slice_mut` works fine, but I'm not sure how to make `multi_slice_move` work:

```markdown
This works:
[`.multi_slice_mut()`]

[`.multi_slice_mut()`]: ArrayBase::multi_slice_mut()


This doesn't:
[`.multi_slice_move()`]

[`.multi_slice_move()`]: ArrayViewMut::multi_slice_move()
```